### PR TITLE
Apply contribution guidelines to "Email" rule

### PR DESCRIPTION
**Pull request by koficodes**
This PR applies the guidelines listed in #921 to Email Validation rule

### CODE REVIEW

File: tests/unit/Rules/EmailTest.php [hunk 1]
```diff
@@ -28,10 +28,14 @@ function class_exists($className)
 
 /**
  * @group  rule
```
Comment (williamespindola) [inline tests/unit/Rules/EmailTest.php]: 🔪 the extra space

File: library/Rules/Email.php [hunk 2]
```diff
@@ -16,13 +16,30 @@
...
     }
 
+    /**
+     * Gets email validator.
+     *
+     * @return EmailValidator | null
+     */
     public function getEmailValidator()
     {
         if (class_exists(EmailValidator::class)
```
Comment (williamespindola) [inline library/Rules/Email.php]: Can you please declare the use of functions `is_string`, `filter_var` and `class_exists`

**Comment by koficodes** [comment]
ok. I will add the requested changes

**Comment by koficodes** [comment]
I have added the requested changes. @williamespindola

File: tests/unit/Rules/EmailTest.php [hunk 3]
```diff
@@ -25,21 +25,24 @@ function class_exists($className)
 
     return \class_exists($className);
 }
-
 /**
  * @group  rule
```
Comment (williamespindola) [inline tests/unit/Rules/EmailTest.php]:
```diff
- * @group  rule
+ * @group rule
```

**Comment by koficodes** [comment]
all good now. @williamespindola

**Comment by koficodes** [comment]
@williamespindola kindly review so I can work on the next PR. Thanks

**Comment by henriquemoody** [comment]
Merged by #1001 with just a few changes.

Thank you very much! 🐼

**Pull request closed without merging**